raphtory/src/core/entities/properties/props.rs

```diff
@@ -202,8 +202,12 @@ impl Meta {
         self.meta_layer.get_name(id)
     }
 
-    pub fn get_node_type_name_by_id(&self, id: usize) -> ArcStr {
-        self.meta_node_type.get_name(id)
+    pub fn get_node_type_name_by_id(&self, id: usize) -> Option<ArcStr> {
+        if self.meta_node_type.has_name(id) && self.meta_node_type.get_name(id) != "_default" {
+            Some(self.meta_node_type.get_name(id))
+        } else {
+            None
+        }
```

Comment:
The assumption is that if you have an internal id, the name entry exists. 

this should be 
```
if id == 0 {
    None
} else {
    Some(self.meta_node_type.get_name(id))
}
```